frontend/src/utilities/segmentIOUtils.tsx

```diff
@@ -84,11 +84,19 @@ export const initSegment = async (props: {
       if (n.parentNode) {
         n.parentNode.insertBefore(t, n);
       }
-      analytics._loadOptions = e;
+      analytics._loadOptions = options;
     };
     analytics.SNIPPET_VERSION = '4.13.1';
     if (segmentKey && enabled) {
-      analytics.load(segmentKey);
+      analytics.load(segmentKey, {
+        cdnURL: 'console.redhat.com/connections/cdn',
+        integrations: {
+          'Segment.io': {
+            apiHost: 'console.redhat.com/connections/api/v1',
+            protocol: 'https',
+          },
+        },
+      });
```

Comment:
@pilhuhn is there anyway I can test this? Or see the backend for requests making it there?

Reply:
Yes. In the browser network tab.
Without the change:
<img width="395" alt="Bildschirmfoto 2024-04-03 um 16 25 55" src="https://github.com/opendatahub-io/odh-dashboard/assets/208246/d8487610-744f-4b58-8284-e5394802ea35">
with the change:
<img width="339" alt="Bildschirmfoto 2024-04-03 um 16 21 29" src="https://github.com/opendatahub-io/odh-dashboard/assets/208246/df44cbe9-ab88-4475-a31a-1cfc12f3fa89">

Note, in dev-mode no analytics is sent out, so those traces would not be in the network tab, despite the console talking about such events.